codec/consolereader.go

```diff
@@ -99,7 +100,7 @@ func (r *ConsoleReader) next(readType int) (out *bstream.Block, err error) {
 	zlog.Debug("next", zap.Int("read_type", readType))
 
 	for line := range r.lines {
-		if !strings.HasPrefix(line, "DMLOG ") {
+		if !strings.HasPrefix(line, "FIRE ") {
```

Comment:
Need and adjustment in the `near-dm-indexer`, @colindickson we will need to sync with this next week (there will be a bunch of flags renaming also).